docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_creating_custom_test.adoc

```diff
@@ -91,6 +91,18 @@ spec:
 . Add your new custom test as an integration test in {ProductName}.
 .. For additional instructions on adding an integration test, see this document: xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[Adding an integration test].
 
+----
```

Comment:
Why is added as a code block and not standard text?

Reply:
This part is not a step of the procedure. I think it should be a additional Note or Tips. I updated to use Note in the PR.